src/utils/ContextSettingsUser.tsx

```diff
@@ -19,8 +19,10 @@ export const UserSettingsContext = createContext<ContextSettingsUserValue>(
 
 const UserSettingsProvider: FC<ContextSettingsUserProviderProps> = ({ children }) => {
   const { isLoginUser } = useContext(GlobalContext);
-  const [defaultWordLanguage, setDefaultWordLanguage] = useState('English');
-  const [defaultWordLanguageTranslate, setDefaultWordLanguageTranslate] = useState('Polish');
+  const [defaultWordLanguage, setDefaultWordLanguage] = useState(mappedLanguages[7].value);
```

Comment:
taka "7" warto wyciągnać gdzieś do zmienn w stylu: `DEFAULT_LANGUAGE_NUMBER`. Wtedy ktoś kto będzie patrzył w kod nie będzie się zastanawiał skąd Ty wytrzasnęłaś tą liczbę tylko nazwa zmiennej od razu mu powie o co chodzi. a jak nie chcesz zmiennej to warto w takich miejscach dodać komentarz